index.html

```diff
@@ -1,21 +1,67 @@
-<!--So, in addition to building an application that calculates daily sales projections for each location (in a file called sales.html), you also need to create a pubic-facing page (in a file called index.html) that is colorful, eye-catching, readable, useful, informative... and ultimately of a quality ready for use on T-shirts, stickers, coffee mugs, cookie bags, napkins, and so on.-->
-
 <!DOCTYPE html>
+
 <html>
-  <head>
-    <meta charset="utf-8">
-    <title></title>
-  </head>
-  <body>
+<head>
+	<meta charset="utf-8">
+	<link href="style.css" rel="stylesheet" type="text/css">
+	<link href="https://fonts.googleapis.com/css?family=Anton|Source+Sans+Pro" rel="stylesheet">
+	<script src="https://use.fontawesome.com/20877e6602.js">
+	</script>
 
-  </body>
-</html>
+	<title>
+	</title>
+</head>
+
+<body id="body">
+	<img alt="Photo of a Salmon" id="salmon" src="img/salmon.png">
+
+	<h1>Pat's Salmon Cookies</h1>
+
+
```

Comment:
You don't need two blank lines between these HTML tags. This is too much whitespace.